pandas/tseries/offsets.py

```diff
@@ -14,6 +14,13 @@
 from pandas._libs import tslib, Timestamp, OutOfBoundsDatetime, Timedelta
 from pandas.util._decorators import cache_readonly
 
+from pandas._libs.tslib import _delta_to_nanoseconds
+from pandas._libs.tslibs.offsets import (  # noqa
+    ApplyTypeError, CacheableOffset,
```

Comment:
you shouldn't need the noqa, that means you are importing extra things that are not needed. IOW some of these are just used internally (now in offsets.pyx)

Reply:
The two unused imports are `WeekDay` and `CacheableOffset`.  They're both used in the tests (though CacheableOffset not really).  I'll update the imports and get rid of the noqa.